scripts/linters/custom_eslint_checks/rules/no-testonly.js

```diff
@@ -32,15 +32,22 @@ module.exports = {
     fixable: null,
     schema: [],
     messages: {
-      notTestOnlyAllowed: 'Can not call a function containing \'testOnly\' ' +
-        'in a JS/TS file that is not a .spec file'
+      notTestOnlyAllowed: 'Please do not call a testOnly function from a ' +
+        'non-test file.'
     }
   },
   create: function(context) {
+    var validateCallExpression = function(node) {
```

Comment:
Do not create another function, it was better before.

Reply:
Done!